src/course-outline/utils.jsx

```diff
@@ -110,7 +110,11 @@ const getHighlightsFormValues = (currentHighlights) => {
 };
 
 const scrollToElement = (ref) => {
```

Comment:
Had a weird bug appear when editing this. Repro steps
1. Add a new section
2. Should scroll to the new section
3. Scroll up to another section and edit its visibility
Expected
- Once everything finishes saving, page should remain on edited section
Actual
- scrolls to the recently added section

This also happens when editing in the basic tab

Reply:
@KristinAoki This should be fixed now. We will only scroll to element when required.